src/cargo/ops/cargo_package.rs

```diff
@@ -151,6 +153,22 @@ pub fn package_one(
     fs::rename(&src_path, &dst_path)
         .with_context(|| "failed to move temporary tarball into final location")?;
 
+    let dst_metadata = dst
+        .file()
+        .metadata()
+        .with_context(|| format!("could not learn metadata for: `{}`", dst_path.display()))?;
+    let compressed_size = dst_metadata.len() as usize;
+
+    let uncompressed = human_readable_bytes(uncompressed_size);
+    let compressed = human_readable_bytes(compressed_size);
+
+    let message = format!(
+        "{} files, {:.1}{}B ({:.1}{}B compressed)",
```

Comment:
Remove these `B`s and we're almost there! 

```suggestion
        "{} files, {:.1}{} ({:.1}{} compressed)",
```

Reply:
Haha, great idea :sweat_smile: